Tensile/LibraryIO.py

```diff
@@ -43,6 +43,12 @@
     print("Message pack python library not detected. Must use YAML backend instead.")
 
 
+def updateProblemDatatypes(problemType):
```

Comment:
nitpick, but not really a fan of this function, since it doesn't get reused and has a very opaque name it's just hiding what's happening. What about having something like

```
for dtype_key in ["DataType", "DestDataType", "ComputeDataType"]:
    solutionState["ProblemType"][dtype_key] = \
                    solutionState["ProblemType"][dtype_key].value
```

Not in a function but just at line 92. That way you're still grouping them, but it's clear what's happening

Reply:
Maybe you just missed it, but this is called in five places across the various functions. I agree it's a bit opaque, but I thought it was the lesser of the two evils compared to the code duplication.

Reply:
Oh I totally did miss that sorry